packages/playground/website/src/lib/hooks.ts

```diff
@@ -40,10 +41,25 @@ export function usePlayground({ blueprint, storage }: UsePlaygroundOptions) {
 			remoteUrl.searchParams.set('storage', storage);
 		}
 
+		const onBlueprintStepError = (step: Step, error: Error) => {
+			if (step.step === 'installPlugin') {
+				// TODO: Open modal to display error message
+				alert(
+					`Failed to install: ${
+						(step.pluginZipFile as CorePluginReference).slug ||
```

Comment:
This type coersion to `CorePluginReference` is questionable - it was a temporary workaround because `step.pluginZipFile` is a `FileReference`, a union of several types which have property `slug`, `name`, `path`, or nothing for the name.